assets/i18n/strings/compile.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @fileoverview Compiles XLB files into a single JSON file with translations. */
+
+const cheerio = require('cheerio');
+const fs = require('fs').promises;
+
+async function main() {
+  const files = await fs.readdir(__dirname);
+  const xlbFiles = files.filter((name) => /\.xlb$/.test(name));
+  const localesPerMessage = {};
+  for (const xlbFile of xlbFiles) {
+    const xml = await fs.readFile(__dirname + '/' + xlbFile, 'utf8');
+    const $ = cheerio.load(xml);
+    let locale = $('localizationbundle').attr('locale').toLocaleLowerCase();
+    if (locale === 'en-gb') {
```

Comment:
What about if locale.startsWith('en-')? Will we have to do the same thing for 'en-us'?

Reply:
We're only expecting this one English variant

Reply:
Checking for all "en-" locales wouldn't hurt though. I just updated the code to do that :)